pkg/test/helpers.go

```diff
@@ -88,7 +88,7 @@ func SetupGH() (client *github.Client, mux *http.ServeMux, serverURL string, tea
 // SeedTestData returns Clients and Informers populated with the
 // given Data.
 // nolint: golint
-func SeedTestData(t *testing.T, ctx context.Context, d Data) (Clients, Informers) {
+func SeedTestData(ctx context.Context, t *testing.T, d Data) (Clients, Informers) {
```

Comment:
I am *meh* on this one… It's a test function, I prefer to have `t *testing.T` at first (makes it easier to detect it's a test function) and put a `nolint` 🙃

Reply:
sgtm